Parkinson/Resources/JSONs/APHTasksAndSchedules.json

```diff
@@ -38,6 +38,22 @@
               
                    },
                    {
+                       "taskTitle" : "PD Questionnaire",
+                       "taskID": "6-PDQ8-20EF83D2-E461-4C20-9024-F43FCAAAF4C8",
+                       "taskFileName" : "PDQ8",
+                       "taskClassName" : "APHPDQ8ViewController",
+                       "taskCompletionTimeString" : "8 Questions"
+                   
+                   },
+                   {
+                       "taskTitle" : "PD Rating Scale",
+                       "taskID": "5-MDSUPDRS-20EF82D1-E461-4C20-9024-F43FCAAAF4C8",
+                       "taskFileName" : "PD_MDRUPDRS",
+                       "taskClassName" : "APHMDRUPDRS",
+                       "taskCompletionTimeString" : "21 Questions"
+                   
```

Comment:
Since the content for these is not present in this version of the codebase, should the schedules be present?